client/components/user/auth/signup/signup.template.html

```diff
@@ -1,57 +1,86 @@
-<div class="container signup">
-  <form role="form" name="signup" method="POST" action="/api/signup" novalidate ng-submit="$ctrl.signup(signup.$valid)">
+<div class="registration container">
```

Comment:
Swap `container` and `registration`. This will prevent Bootstrap from accidentally overriding your styles.